cdhweb/people/templates/people/profile.html

```diff
@@ -20,7 +20,7 @@ <h1>{{ page.title }}</h1>
 <div class="links">
     {# NOTE equiv to contributors in project #}
     <ul>
-    {% for link in person.related_links.all %}
+    {% for link in page.person.related_links.all %}
```

Comment:
i figured out you can actually set `context_object_name` on wagtail pages after all; we should probably do that so this is `profile.person.related_links`, etc. I have set it for `Event` and `Project` in my branch for events

Reply:
do you want me to make that change now? or would it make sense to fold it into your branch?

Reply:
probably more sense in my branch tbh. i'll just rebase as soon as we merge this and then make the change